### ISSUE TITLE
[BUG] Breaking reformat in in-line conditional in class attribute

### ISSUE THREAD

**Issue by dtcooper** [Riverside-Healthcare/djLint#28]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Debian 11 — python:3.9 docker container
 - Python Version: 3.9 (``python --version``)
 - djLint Version:  0.4.2

## Issue + How To Reproduce

Hi there,

Using inline conditionals in the `class="..."` attribute of a tag we have the following.

```jinja2
<div class="class1{% if condition %} class2{% endif %}">content</div>
                                    ^ single space significant
```

Running `djlint --reformat` on the file, we get,

```jinja2
<div class="class1{% if condition %}class2{% endif %}">content</div>
                                    ^ single space removed
```

A workaround would be to put the space _before_ the conditional, ie,

```jinja2
<div class="class1 {% if condition %}class2{% endif %}">content</div>
                  ^ space refactored to be here
```

Is this intended? Should I apply this workaround throughout my codebase before reformatting?

In any event, thanks for the great tool!

Cheers,

David

**Comment by christopherpickering** [comment]
Hey David, thanks for the report! I understand the importance of this issue.

I think your recommended workaround is the best solution for now, I've done the same in my code. 

The challenge is knowing when the space is intentional vs not intentional. For example,
```html
<a class="{% if condition %}
                 stuff
                {% endif %}">
```
has a bunch of random spaces, and potentially tabs, that should be reformatted.

I'll leave the issue open for now in case another idea pops up!

**Comment by christopherpickering** [comment]
For now I've added this to "best practices" in the docs. http://djlint.com/djlint/best_practices.html
